src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java

```diff
@@ -210,4 +188,33 @@ void exportSelected(ActionEvent event) {
        viewModel.startExport();
     }
 
+    private JabRefIcon getFileFilterIcon(FileChooser.ExtensionFilter fileFilter) {
```

Comment:
The view class shouldn't contain any logic, so all these 3 methods here should go to appropriate view model classes (probably a ExtensionFilterViewModel needs to be introduced for this).

Reply:
Hm. I don't see any reason for creating a viewModel for this, seems a bit overblown

Reply:
It's similar to the FileNodeViewModel. It's a bit of an advantage of the MVVM pattern that one usually has wrappers for many logic classes. 

How would you write a test for the getFileFilterIcon method otherwise?